phpstan.neon

```diff
@@ -8,7 +8,17 @@ parameters:
          - src
          - tests
     reportUnmatchedIgnoredErrors: false
+    excludePaths:
+         - '*tests/*/Fixture/*'
+         - '*tests/*/Source/*'
     ignoreErrors:
+        - "#^Parameter \\#1 \\$repositoryName of static method TYPO3\\\\CMS\\\\Core\\\\Utility\\\\ClassNamingUtility\\:\\:translateRepositoryNameToModelName\\(\\) expects class\\-string\\<TYPO3\\\\CMS\\\\Extbase\\\\Persistence\\\\RepositoryInterface\\>, class\\-string given\\.$#"
+        - "#^Parameter \\#1 \\$repositoryName of static method TYPO3\\\\CMS\\\\Core\\\\Utility\\\\ClassNamingUtility\\:\\:translateRepositoryNameToModelName\\(\\) expects class\\-string\\<TYPO3\\\\CMS\\\\Extbase\\\\Persistence\\\\RepositoryInterface\\>, string given\\.$#"
+        - "#^Call to static method resolve\\(\\) on an unknown class TYPO3\\\\CMS\\\\Extbase\\\\Validation\\\\ValidatorClassNameResolver\\.$#"
         -
             message: '#^Class TYPO3\\CMS\\Core\\Context\\[a-zA-Z]* not found\.#'
             path: %currentWorkingDirectory%/src/Type/ContextDynamicReturnTypeExtension.php
+        -
+            message: "#^Calling PHPStan\\\\Reflection\\\\InitializerExprTypeResolver\\:\\:getClassConstFetchType\\(\\) is not covered by backward compatibility promise\\. The method might change in a minor PHPStan version\\.$#"
+            count: 1
+            path: %currentWorkingDirectory%/src/Rule/ValidatorResolverOptionsRule.php
```

Comment:
```suggestion
            path: src/Rule/ValidatorResolverOptionsRule.php
```
%currentWorkingDirectory% should not be used anymore as paths are now always relative to the configuration file and it'll fail anyways if you are executing phpstan form a different folder (working directory). So we should also remove it in the existing line.